geoportailv3/templates/index.html

```diff
@@ -196,7 +197,9 @@ <h2 translate>infos</h2>
     <script src="${request.route_url('deps.js')}"></script>
     <script src="${request.static_url('geoportailv3:static/js/main.js')}"></script>
     <script src="${request.static_url('%s/ngeo/utils/watchwatchers.js' % node_modules_path)}"></script>
+    <script src="${request.static_url('%s/proj4/dist/proj4-src.js' % node_modules_path)}"></script>
 % else:
+    <script src="${request.static_url('%s/proj4/dist/proj4-src.js' % node_modules_path)}"></script>
```

Comment:
It is not :-)

The `build.js` file located in `geoportailv3/static/build`, and loaded by the page in non-debug mode, is a concatenation of all the js builds. More specifically it's a concatenation of `jquery.min.js`, `angular.min.js`, `angular-animate.min.js`, `angular-gettext.min.js`, `bootstrap.min.js`, and `.build/build.js`, which is the app + ol3 + ngeo + closure Closure-compiled code. And soon, it will also include the application's directive partials (https://github.com/Geoportail-Luxembourg/geoportailv3/pull/293). I hope this clarifies.